integration/tests/runtime-workflow-postgres.ts

```diff
@@ -283,6 +302,26 @@ test('Find all notes', async () => {
   })
 })
 
+test('Find all notes with count', async () => {
+  const { data, errors } = await client.query({
+    operationName: 'findNotes',
+    query: FIND_NOTES_WITH_COUNT
+  })
+
+  expect(data).toBeDefined()
+  expect(data.findNotes.count).toBeDefined()
```

Comment:
If `count==null` this will be `true`, while in reality the value is not sent. Maybe we should check for the actual value.

Reply:
Definitely.